resources/js/beatmapsets-show/info.tsx

```diff
@@ -191,14 +196,14 @@ export default class Info extends React.Component<Props> {
             </div>
           </div>
 
-          {tags.length > 0 &&
+          {this.tags.length > 0 &&
             <div className='beatmapset-info__row beatmapset-info__row--value-overflow'>
               <h3 className='beatmapset-info__header'>
                 {trans('beatmapsets.show.info.tags')}
               </h3>
               <div className='beatmapset-info__value-overflow'>
-                {tags.map((tag, i) => (
-                  <React.Fragment key={`${tag}-${i}`}>
+                {this.tags.map((tag) => (
+                  <React.Fragment key={`${tag}`}>
```

Comment:
the `-${i}` is required to avoid duplicate keys (mainly for mapper tags but also now in conjunction with user tags)